ethcore/res/ethereum/tests-issues/currents.json

```diff
@@ -0,0 +1,215 @@
+{ "block":
+        [
+
+                {
+                        "reference": 9568,
+                        "failing": "stCreateTest",
+                        "subtests": [
+                                "CreateOOGafterInitCodeRevert_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata3_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeRevert2_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCode_d0g1v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndataSize_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCode_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata2_d0g0v0_Constantinople",
```

Comment:
Please cherry-pick https://github.com/paritytech/parity-ethereum/pull/9577/commits/544f6a3d0c9fa61f626b8bb3d137f7c5039d7af7
I think the only one left here right now should be `CreateOOGafterInitCodeReturndata2`.